addons/mail/models/mail_thread.py

```diff
@@ -430,8 +430,8 @@ def _prepare_tracking(self, fields):
         fnames = self._get_tracked_fields().intersection(fields)
         if not fnames:
             return
-        func = self.browse()._finalize_tracking
-        [initial_values] = self.env.cr.precommit.add(func, dict)
+        self.env.cr.precommit.add(self._finalize_tracking)
+        initial_values = self.env.cr.precommit.data.setdefault('mail_tracking.%s' % self._name, {})
```

Comment:
I would rename this into mail.tracking_%s to follow the convention of namespacing by module name

Reply:
Considering that `_name` will have dots, should probably use them all the way then. 
`mail.tracking.account.move` vs `mail.tracking_account.move`